### PR TITLE
Nextflow upgrades

### PR DESCRIPTION
**Pull request by NickSwainston**
- Upgraded to DSL 2
- Removed some redundant scripts
- Added automated versioning
- Publish data to the output directory
- Added a --help to describe the inputs
- Updated the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The workflow is described in [Hancock et al. 2018](https://ui.adsabs.harvard.edu
 - Persistent source catalogue creation:
   - Stack the warped images into a cube and form a mean image
   - Source find on the mean image to make a master catalogue
-  - Priorized fit this catalogue into each of the individual images
+  - Prioritized fit this catalogue into each of the individual images
```
Comment (PaulHancock) [inline README.md]: Wasn't a typo!

File: robbie.nf [hunk 2]
```diff
@@ -0,0 +1,486 @@
...
+             |                Use a source finding file. [default: ${params.use_region_file}]
+             | --region_file  The location of the source finding file. [default: ${params.region_file}]
+             |
+             |Directory arguments:
+             |  --output_dir  The directory to output the results to.
+             |                [default: ${params.output_dir}]
+             |  -keep_epoch_images
+             |                Keep the epoch images after warping.
+             |                [default: ${params.keep_epoch_images}]
+             |  -w            The Nextflow work directory. Delete the directory once the processs
```
Comment (PaulHancock) [inline robbie.nf]: Should these options be have double dashes for them? Nextflow treats single/double dashes differently and I often get it wrong.

**Review by PaulHancock — APPROVED** [review]

File: robbie.nf (hunk 2 above)
Comment (NickSwainston) [reply robbie.nf]: Yup you're right

File: README.md (hunk 1 above)
Comment (NickSwainston) [reply README.md]: fixed

**Pull request merged**